server/utils/assessments/daysUntilDue.test.ts

```diff
@@ -0,0 +1,21 @@
+import { addBusinessDays as addBusinessDaysDateFns, formatISO } from 'date-fns'
+import { assessmentSummaryFactory } from '../../testutils/factories'
+import { daysUntilDue } from './dateUtils'
+
+jest.mock('../dateUtils', () => ({
+  DateFormats: {
+    isoToDateObj: jest.fn().mockImplementation(date => new Date(date)),
+    dateObjToIsoDate: jest.fn().mockImplementation(date => formatISO(date)),
+    dateObjToIsoDateTime: jest.fn().mockImplementation(date => formatISO(date)),
+    differenceInBusinessDays: jest.fn().mockReturnValue(1),
+  },
+  addBusinessDays: jest.fn().mockImplementation((date, days) => addBusinessDaysDateFns(date, days)),
```

Comment:
I feel like this might be easier to follow if we did the mocking in the test itself, rather than the setup here. Would also be good to confirm that `isoToDateObj` and `addBusinessDays` were called with the arguments we expect too.

Reply:
Yeah good shout. I don't know if it's easier to follow but it's definitely a more thorough test now.